src/parquet/column/serialized-page.cc

```diff
@@ -45,23 +45,36 @@ SerializedPageReader::SerializedPageReader(std::unique_ptr<InputStream> stream,
   }
 }
 
-// TODO(wesm): this may differ from file to file
-static constexpr int DATA_PAGE_SIZE = 64 * 1024;
+static constexpr int PAGE_SIZE = 64 * 1024;
```

Comment:
As part of making the max page serialized page header size configurable, rename this to `DEFAULT_MAX_PAGE_HEADER_SIZE` and use that as the default for the configurable variable.